components/collection/drop/HolderOfGenerative.vue

```diff
@@ -16,7 +16,8 @@
       @close="isSuccessModalActive = false">
       <CollectionDropModalSharedSuccessfulDrop
         v-if="claimedNft"
-        :can-list-nft="canListMintedNft"
+        :minting-session="mintingSession"
+        :can-list-nfts="canListMintedNft"
```

Comment:
`CollectionDropModalSharedSuccessfulDrop` is used inside `DropConfirmModal` where passed slightly different prop values since it's using `claimedNft` and it's not exactly the same as the other drop mints 

`PaidMint.vue` it's using the massmint variant

Reply:
i don't think i follow your drift

whatever compoennt it is, why can't it read from store

why store here and then turn it into props